libs/domains/services/feature/src/lib/pods-metrics/pods-metrics.tsx

```diff
@@ -9,60 +28,256 @@ export interface PodsMetricsProps {
+
+          if (stateA === 'ERROR' && stateB === 'ERROR') {
+            return 0
+          } else if (stateA === 'ERROR' && stateB !== 'ERROR') {
+            return -1
+          } else if (stateA !== 'ERROR' && stateB === 'ERROR') {
+            return 1
+          } else if (stateA === 'WARNING' && stateB === 'WARNING') {
+            return 0
+          } else if (stateA === 'WARNING' && stateB !== 'WARNING') {
+            return -1
+          } else if (stateA !== 'WARNING' && stateB === 'WARNING') {
+            return 1
+          } else {
+            return stateA.localeCompare(stateB)
+          }
+        },
+      }),
+      ...(service?.serviceType === 'DATABASE'
+        ? []
+        : [
+            columnHelper.accessor('service_version', {
+              header: 'Version',
+              cell: (info) => {
+                const value = info.getValue()
+                return (
+                  value && (
+                    <Badge variant="surface" size="xs" className="gap-2">
+                      {containerImage ? (
+                        `${containerImage.image_name}:${containerImage.tag}`
+                      ) : (
+                        <>
+                          <Icon name={IconAwesomeEnum.CODE_COMMIT} />
+                          {value.substring(0, 7)}
+                        </>
+                      )}
+                    </Badge>
+                  )
+                )
+              },
+            }),
+          ]),
+      columnHelper.accessor('memory.current_percent', {
+        header: 'Memory',
+        cell: (info) => (info.getValue() !== undefined ? `${info.getValue()}%` : placeholder),
+      }),
+      columnHelper.accessor('cpu.current_percent', {
+        header: 'vCPU',
+        cell: (info) => (info.getValue() !== undefined ? `${info.getValue()}%` : placeholder),
+      }),
+      ...(service?.serviceType === 'JOB'
+        ? []
+        : [
+            columnHelper.accessor('storages', {
+              header: 'Storage',
+              cell: (info) => {
+                const value = info.getValue()
+                return value?.length
+                  ? // https://qovery.slack.com/archives/C02NQ0LC8M9/p1693235796272359
+                    `${Math.max(...value.map(({ current_percent }) => current_percent))}%`
+                  : '-'
+              },
+            }),
+          ]),
+      columnHelper.accessor('started_at', {
+        header: match(service)
+          .with({ serviceType: ServiceTypeEnum.JOB }, () => 'Start time')
+          .otherwise(() => 'Age'),
+        cell: (info) => {
+          const value = info.getValue()
+          return value ? (
+            <Tooltip content={dateFullFormat(value)}>
+              <span className="text-xs text-neutral-350">{timeAgo(new Date(value))}</span>
+            </Tooltip>
+          ) : (
+            placeholder
+          )
+        },
+      }),
+    ],
+    [service, containerImage]
```

Comment:
This is not needed because columnHelper and placeholder are constant.
https://github.com/Qovery/console/pull/894/commits/1f600306965fd6e114ae0ccae2dd16598a3ca43e